p2p/router.go

```diff
@@ -207,6 +228,11 @@ func NewRouter(
 	transports []Transport,
 	options RouterOptions,
 ) (*Router, error) {
+
+	if options.QueueType == "" {
+		options.QueueType = os.Getenv("TM_P2P_QUEUE")
```

Comment:
Instead of reading from the environment variable here, why not just default to FIFO?

Reply:
the way the router is constructed in the node doesn't actually pass the options struct through, so we'd need some more plumbing.

Reply:
Ahh, can we update `node.go` then instead?

Reply:
definitely, though putting it here, means that we can test it in this package, which is nice, though probably not worth it.